plugins/test_control_plugin/test_control_plugin.cpp

```diff
@@ -26,6 +26,7 @@ class test_control_plugin_impl {
    account_name        _producer;
    int32_t             _where_in_sequence;
    int32_t             _producer_sequence;
+   uint32_t            _first_sequence_timeslot;
```

Comment:
All three of these ints are not initialized and looks like they are accessed before being assigned causing undefined behavior.